RHIDP-12206 - Disable the Global Floating Action Button (#2119)
redhat-developer/red-hat-developers-documentation-rhdh: assemblies/configure_customizing-rhdh/assembly-configure-a-floating-action-button-in-rhdh.adoc

:context: configure-a-floating-action-button-in-rhdh

[role="_abstract"]
Configure any action as a floating button in the {product-short} instance by using the floating action button plugin.

[IMPORTANT]
====

> As of {product-very-short} 1.9, the Global Floating Action Button plugin is disabled by default and is deprecated. The Global Header plugin provides equivalent and improved navigation capabilities by using quick links and starred items.

Reviewer: If you want to embed a link to the deprecation notice - it's here in the release notes doc: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.9/html/red_hat_developer_hub_release_notes/deprecated-features

> As of {product-very-short} 1.9, the Global Floating Action Button plugin is disabled by default and is link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.9/html/red_hat_developer_hub_release_notes/deprecated-features[deprecated]. The Global Header plugin provides equivalent and improved navigation capabilities by using quick links and starred items.

Reviewer 2: ```suggestion
As of {product-very-short} 1.10, the Global Floating Action Button plugin is disabled by default and is link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.9/html/red_hat_developer_hub_release_notes/deprecated-features[deprecated]. The Global Header plugin provides equivalent and improved navigation capabilities by using quick links and starred items.
```

Hi @Gerry-Forde , the FAB plugin is enabled by default for 1.9. So not sure why in 1.9 release notes it states deprecated for FAB. Better to confirm further on it.

> For new deployments, use the Global Header instead. See xref:configure-the-global-header-in-rhdh_customizing-rhdh[Configure the Global Header in {product}]. 

Reviewer 3: ```suggestion
{product-very-short} 1.10 disables the Global Floating Action Button plugin by default. For new deployments, use the Global Header plugin, which uses quick links and starred items to improve navigation.
```
(As per the vale suggestion, we will need to move the links to the "additional resources" section at the end of the assembly)

> Existing deployments that explicitly enable the Global Floating Action Button are not affected and can continue using this plugin.  

Reviewer: Would it be customary to mention at the end here that while they can continue to use it, since it is deprecated, it will be removed in the future, and they may need to adjust their configuration in future versions?

> Existing deployments that explicitly enable the Global Floating Action Button are not affected and you can continue to use this plugin. However, as the plugin is deprecated, it will be removed in a future {product-very-short} version, and you might then need to adjust your configuration. 

Reviewer 3: ```suggestion
Existing deployments that explicitly enable the Global Floating Action Button remain functional. However, as the plugin is deprecated, it will be removed in a future {product-very-short} version, and you must eventually adjust your configuration. 
```
(Minor suggestions. Please ignore if you feel they don't apply!)

> Existing deployments that explicitly enable the Global Floating Action Button are not affected and you can continue to use this plugin. However, as the plugin is deprecated, it will be removed in a future {product-very-short} version, and you might need to adjust your configuration in future versions. 

Reviewer: Nitpicking, it reads a bit redundant to me to say "**in a future RHDH version**, and you might need to adjust your configuration **in future versions**."

Something like swapping the second mention with "then" or "when that happens" could sidestep the repetition, but I'm happy with whatever you think is best for that sentence. I like the overall updates in this PR, and I think adding the link above is helpful. 

It would be good to still get a more technical review, if possible.
